### PR TITLE
Add a bubble effect for professionnal list and services list

### PR DESCRIPTION
**Pull request by ThibaultBardinetLanglois**

### CODE REVIEW

**Review by ced37 — DISMISSED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]

File: templates/client/list.html.twig [hunk 1]
```diff
@@ -5,20 +5,10 @@
...
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-                <li></li>
-            </ul>
+            {#
+            {% for service in services %}
+                < class="bubble-service"><p>{{ service.name }}</p></>
+            {% endfor %} #}
```
Comment (RomainClair) [inline templates/client/list.html.twig]: You shouldn't commit dead code

**Review by RomainClair — CHANGES_REQUESTED** [review]
There probably shouldn't be a Version file if no Entity are in the PR

File: templates/client/list.html.twig (hunk 1 above)
Comment (ThibaultBardinetLanglois) [reply templates/client/list.html.twig]: Ok this is because this pull request anticipate the next one i have just send.

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by ced37 — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
There should not be a Version* file in a PR if there's no new entity or a change in one.

**Comment by ThibaultBardinetLanglois** [comment]
I don't know what is a Version*file, maybe a migration file?

**Comment by ThibaultBardinetLanglois** [comment]
Ok sorry i understand now that Version*file are in the migrations folder, to be honest i don't know this principe, i just push my branch, nothing else

**Review by ced37 — DISMISSED** [review]

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by floriane-cheneveau — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
The migration file (A src/migrations/Version***.php file is still in this PR with no entity.
It means that it will try to create/alter the database but it shouldn't because no new entity (or entity modifications) means no changes in the database.

**Review by StephaneSauvaget — DISMISSED** [review]
Please, i'm sure it's ok !

**Review by ced37 — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
This PR delete the previous (from the dev branch) migration file but the entities are still there

**Review by StephaneSauvaget — DISMISSED** [review]

**Review by ced37 — DISMISSED** [review]

**Review by RomainClair — COMMENTED** [review]
Restore the migration files according to the dev branch
It should be because this branch doesn't change entities and the dev branch is in a good state for migrations/entity

**Review by StephaneSauvaget — APPROVED** [review]
Thanks, romain

**Review by ced37 — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**